addons/web/static/src/scss/search_panel.scss

```diff
@@ -15,6 +15,7 @@ $o-searchpanel-filter-default-color: #D59244;
         flex: 1 1 100%;
         overflow: auto; // make the renderer and search panel scroll individually
         max-height: 100%;
+        height: 100%;
```

Comment:
do you need this?

Reply:
It's for the search panel to take the full height:
 - Without: https://drive.google.com/open?id=1Av4ivMuoNbLLXlUPZFVsRc2HdooA7j8T
 - With: https://drive.google.com/open?id=18Fr-d7ItSmBcvHZsx1Kkv1GmvsgbFpgq

Reply:
I can't reproduce on the last v13 runbot...

Reply:
Sorry I mixed multiple things... It was rather to fix this issue: https://drive.google.com/open?id=1q0X7qKpTqeO_XC4NhiuES6z2vrs2WMcu but it seems it's no longer an issue on the latest runbot!

My bad, I'll remove the height: 100% here